src/nu/xom/tests/BuilderTest.java

```diff
@@ -1840,7 +1863,20 @@ public void testBaseRelativeResolutionRemotelyWithDirectory()
     // and it is fixed in Xerces 2.6.
     public void testRelativeURIResolutionAgainstARedirectedBase()
       throws IOException, ParsingException {
-        builder.build("http://www.ibiblio.org/xml/redirecttest.xml");
+        if (CITestUtil.isRunningInCI()) {
+            // Skip network tests in CI where connectivity may be unreliable
+            return;
+        }
+        try {
+            builder.build("http://www.ibiblio.org/xml/redirecttest.xml");
+        } catch (IOException e) {
+            if (CITestUtil.isNetworkException(e)) {
```

Comment:
we don't need this and the if (CITestUtil.isRunningInCI()) check above. I thijk keep this one and remove that one.